core/src/commonTest/kotlin/com/xebia/functional/xef/auto/CoreAIScopeSpec.kt

```diff
@@ -0,0 +1,113 @@
+package com.xebia.functional.xef.auto
+
+import com.xebia.functional.tokenizer.ModelType
+import com.xebia.functional.xef.data.TestEmbeddings
+import com.xebia.functional.xef.data.TestModel
+import com.xebia.functional.xef.vectorstores.ConversationId
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.ints.shouldBeLessThan
+import io.kotest.matchers.shouldBe
+import kotlinx.uuid.UUID
+import kotlinx.uuid.generateUUID
+
+class CoreAIScopeSpec :
```

Comment:
The CoreAiScope is renamed to `Conversation` and the `ai` block to `conversation` in #311

Reply:
I'll change it after merging your PR into `main`

Reply:
merged!